### PR TITLE
Update AFE paragraph display on international Dance Party pages

### PR DESCRIPTION
**Pull request by clareconstantine**
On the Dance Party page in non-English languages, we had a blurb about AFE that wasn't very visually engaging. Now we have the same content displayed in a block with an image.

**Non-English:** (I don't have the translations locally, but I confirmed this is the correct string key in i18n)

Screen Shot 2020-09-21 at 10 15 03 AM

**English:**

Screen Shot 2020-09-21 at 10 21 52 AM

## Links

- [spec]()
- [jira]()

## Testing story

# Reviewer Checklist:

- [ ] Tests provide adequate coverage
- [ ] Code is well-commented
- [ ] New features are translatable or updates will not break translations
- [ ] Relevant documentation has been added or updated
- [ ] User impact is well-understood and desirable
- [ ] Pull Request is labeled appropriately
- [ ] Follow-up work items (including potential tech debt) are tracked and linked

### CODE REVIEW

**Comment by tess323** [comment]
Looks great to me!

File: pegasus/sites.v3/code.org/views/dance_sponsor_logo.haml [hunk 1]
```diff
@@ -1,9 +1,27 @@
...
+  %link{href: "/css/full-resource-block.css", rel: "stylesheet"}
+  %link{href: "/css/tools.css", rel: "stylesheet"}
+
+  .sponsor-logo-with-text
+    = I18n.t(:hoc2018_dance_amazon_sponsor_section_with_logo_title)
+    .sponsor-logo-image
+      %img{src:"/images/fit-250/AFE-dark-logo.png"}
+
+  .full-resource-block{style: "margin-top: 20px;"}
+    %a{href: "https://www.amazonfutureengineer.com", target: "_blank"}
```
Comment (cforkish) [inline pegasus/sites.v3/code.org/views/dance_sponsor_logo.haml]: tiny NIT: now that we're using this url twice, i'd consider putting it in a constant.

**Review by cforkish — APPROVED** [review]
LGTM!

**Review by tess323 — APPROVED** [review]

**Review by jmkulwik — APPROVED** [review]
Looks good!

File: pegasus/sites.v3/code.org/views/dance_sponsor_logo.haml (hunk 1 above)
Comment (clareconstantine) [reply pegasus/sites.v3/code.org/views/dance_sponsor_logo.haml]: added! thanks!

**Pull request merged**